locales/en.yml

```diff
@@ -75,7 +75,9 @@ en:
 
           Service:
               A service provided by sccli or systemd. For example:
-               docker, k8s, or openshift etc.
+               sccli service like docker, k8s, openshift etc
```

Comment:
Currently `sccli` able to understand docker, kubernetes and openshift. It would be nice if you folks can try latest version of adb-utils which have improved sccli.